sjb/config/test_cases/test_branch_origin_extended_conformance_gce.yml

```diff
@@ -169,6 +169,7 @@ rectory to run."
                 artifact_dir="${base_artifact_dir}/nodes/${name}"
                 mkdir -p "${artifact_dir}" "${artifact_dir}/generated" "${artifact_dir}/journals"
                 gcloud compute ssh "${instance}" -- sudo journalctl --unit origin-node.service --no-pager --all --lines=all 2>&1 > "${artifact_dir}/journals/origin-node.service"
+                gcloud compute ssh "${instance}" -- sudo journalctl --unit dnsmasq.service --no-pager --all --lines=all 2>&1 > "${artifact_dir}/journals/dnsmasq.service"
```

Comment:
Masters are also nodes -- shove this below the `if/elif/else`.

Also add `|| true` to the end of the line please